maintainers/maintainer-list.nix

```diff
@@ -2965,6 +2965,12 @@
     githubId = 1743184;
     name = "Boris Babić";
   };
+  bornedj = {
+    email = "borne.danielj@gmail.com";
+    github = "bornedj";
+    githubId = 65796927;
+    name = "Daniel Borne";
+  };
```

Comment:
This must go in a separate commit that comes before the package, but you should keep it in this PR.